src/helpers/errorHelpers.js

```diff
@@ -0,0 +1,15 @@
+const errorHelpers = {
+  /**
+   * @description Handles 404 errors
+   * @param  {object} req The HTTP request object
+   * @param  {object} res The HTTP response object
+   * @param  {string} item The item not found
+   * @returns {object} The response object
+   */
+  notFound: (req, res, item) =>
+    res.status(404).jsend.fail({
```

Comment:
I think this `jsend` might lead to bugs on your code, as I do not find any `jsend` in your dependency. However, I do not think you should generate your response since we already have a responseGenerator. You can find that here: https://github.com/andela/ah-kifaru-backend/blob/develop/src/helpers/responseGenerator.js